packages/core/data-transfer/src/strapi/remote/handlers/push.ts

```diff
@@ -452,6 +452,11 @@ export const createPushController = handlerControllerFactory<Partial<PushHandler
       getStrapi: () => strapi as LoadedStrapi,
     });
 
+    this.provider.onWarning = (message) => {
+      // TODO send a warning message to the client
```

Comment:
```suggestion
      // send a warning message to the client
```

Reply:
But we need to implement that in the following task, this only prints a message in the server

Reply:
Sorry, missed the word "client"!